TrkHitReco/src/FlagBkgHits_module.cc

```diff
@@ -52,6 +56,7 @@ namespace mu2e
         fhicl::OptionalTable<TNTClusterer::Config>    TNTClustering{        Name("TNTClustering"),        Comment("TNT Clusterer config") };
         fhicl::OptionalTable<Chi2Clusterer::Config>   Chi2Clustering{       Name("Chi2Clustering"),       Comment("Chi2 Clusterer config") };
         fhicl::Atom<std::string>                      kerasWeights{         Name("KerasWeights"),         Comment("Weights for keras model") };
+        fhicl::Atom<bool>                             useSLine{             Name("UseSLine"),             Comment("Use SLine info") };
         fhicl::Atom<float>                            kerasQuality{         Name("KerasQuality"),         Comment("Keras quality cut") };
       };
```

Comment:
Would it make sense to add a config for the ComboHit level of the Input collection?  Since that could determine what training to use?  Also you could check that the input collection actually has the expected level, and throw an exception if not.

Reply:
Do you want to make it so that user don't choose the training specifically but rather specifies the clusterer and input level of the CH collection? I think it makes sense since with the combination of clusterers, trainings and inputs, there is an increasing probability for user error.